components/dashboards-web-component/src/designer/DashboardDesigner.jsx

```diff
@@ -1,31 +1,30 @@
 /*
  *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
  *
- *  WSO2 Inc. licenses this file to you under the Apache License,
- *  Version 2.0 (the "License"); you may not use this file except
- *  in compliance with the License.
- *  You may obtain a copy of the License at
+ * WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Please revert changes in the license header

Reply:
changed with 910980c413d9aea940c47a751f949593c1fc5ec0